.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+  "presets": ["es2015"],
```

Comment:
I think Babel defaults to this.

Reply:
I think prob so.  I can check and remove if it's redundant.